playbooks/roles/openvpn/handlers/main.yml

```diff
@@ -1,5 +1,8 @@
 ---
 - name: Restart OpenVPN
-  service:
-    name: openvpn
+  systemd:
+    name: "{{ item }}"
     state: restarted
+  with_items:
```

Comment:
The two service names in this `with_items` are duplicated in a couple other places. Can you move them into a var in `playbooks/roles/openvpn/vars/`?